src/applications/financial-status-report/wizard/WizardContainer.jsx

```diff
@@ -7,9 +7,8 @@ import {
   DowntimeNotification,
   externalServices,
 } from 'platform/monitoring/DowntimeNotification';
-import Wizard, {
-  WIZARD_STATUS_COMPLETE,
-} from 'applications/static-pages/wizard';
+import Wizard from 'applications/static-pages/wizard';
```

Comment:
Is this `Wizard` also available in `platform/site-wide/wizard` ?

Reply:
No I dont think so, was getting strange JS errors when using it in Yarn

Reply:
Sounds good!